dashboard/app/models/course_offering.rb

```diff
@@ -104,8 +104,17 @@ def any_version_is_assignable_editor_experiment?(user)
     course_versions.any? {|cv| cv.content_root.is_a?(Unit) && cv.has_editor_experiment?(user)}
   end
 
+  def self.all_course_offerings
+    if should_cache?
+      @@course_offerings ||= CourseOffering.all.includes(course_versions: :content_root)
+      @@course_offerings
```

Comment:
nit: i don't think this line is needed, since the one above should return the same thing?